src/PKSim.Core/Mappers/ProcessToProcessBuilderMapper.cs

```diff
@@ -294,7 +294,7 @@ private ReactionBuilder createReactionFromProcess(CompoundProcess process, strin
       public TransportBuilder PassiveTransportProcessFrom(CompoundProcess compoundProcess, string compoundName, IFormulaCache formulaCache)
       {
          var passiveProcess = _cloner.Clone(_simulationActiveProcessRepository.ProcessFor<PKSimTransport>(compoundProcess.InternalName));
-         passiveProcess.Name = compoundProcess.Name;
+         passiveProcess.Name = CompositeNameFor(compoundProcess.Name, compoundName);
```

Comment:
@Yuri05 I am making the name of the process unique. This bug IS in v11

Reply:
This is better for sure than having several processes with the same name :)